server/src/test/java/org/apache/druid/server/http/DataSourcesResourceTest.java

```diff
@@ -268,11 +268,11 @@ public Access authorize(AuthenticationResult authenticationResult1, Resource res
 
     Assert.assertEquals(200, response.getStatus());
     Assert.assertEquals(1, result.size());
-    Assert.assertEquals(
+    TestUtils.assertEqualsImmutableDruidDataSource(
         listDataSources.subList(0, 1).stream()
                        .map(DruidDataSource::toImmutableDruidDataSource)
-                       .collect(Collectors.toSet()),
-        new HashSet<>(result)
+                       .collect(Collectors.toList()),
```

Comment:
Same here. They should be sorted first.

Reply:
Fixed. ImmutableDruidDataSourceTestUtils.assertEqualsImmutableDruidDataSource(List<>, List<> ) takes care of this as explained in comment https://github.com/apache/incubator-druid/pull/7933#discussion_r302680587.